recordings/basic-setup/run.sh

```diff
@@ -0,0 +1,292 @@
+sleep-extra 1
+run <<'EOF'
+kubectl apply --server-side -f=./examples/common/cert-manager.yaml
+EOF
+run <<'EOF'
+kubectl -n=prometheus-operator apply --server-side -f=./examples/third-party/prometheus-operator/
+EOF
+run <<'EOF'
+kubectl wait --for='condition=established' crd/certificates.cert-manager.io crd/issuers.cert-manager.io
+EOF
+run <<'EOF'
+kubectl wait --for='condition=established' \
+crd/prometheuses.monitoring.coreos.com \
+crd/prometheusrules.monitoring.coreos.com \
+crd/servicemonitors.monitoring.coreos.com
+EOF
+run <<'EOF'
+kubectl -n=cert-manager rollout status --timeout=10m deployment.apps/cert-manager{,-cainjector,-webhook}
+EOF
+run <<'EOF'
+kubectl -n=prometheus-operator rollout status --timeout=10m deployment.apps/prometheus-operator
+EOF
+
+run <<< "# Now let's deploy Scylla Operator."
+run <<'EOF'
+kubectl -n=scylla-operator apply --server-side -f=./deploy/operator/
+EOF
+run <<'EOF'
+kubectl wait --for='condition=established' \
+crd/scyllaclusters.scylla.scylladb.com \
+crd/nodeconfigs.scylla.scylladb.com \
+crd/scyllaoperatorconfigs.scylla.scylladb.com \
+crd/scylladbmonitorings.scylla.scylladb.com
+EOF
+run <<'EOF'
+kubectl -n=scylla-operator rollout status --timeout=10m deployment.apps/{scylla-operator,webhook-server}
+EOF
+
+run <<< "# To run ScyllaClusters on local disks (NVMes in production), we need to set up our storage."
+
+run <<< "# We need to set up our local disks into an XFS directory, which will be used by ScyllaDB local CSI driver to provision PVCs."
+run <<< "# We will also enable node tuning."
+
+run <<< "# Create a NodeConfig to set up a RAID array, a filesystem on top of it, and to tune it."
+run <<< "# Note that this part depends on your infrastructure, disk names, and other properties."
+run <<< "# Be sure to double check the NodeConfig and adjust it, if needs be."
+run <<'EOF'
+yq -e eval --colors '.' ./examples/gke/nodeconfig-alpha.yaml | head -n 25
+EOF
+sleep-extra 10
+run <<'EOF'
+kubectl apply --server-side -f=./examples/gke/nodeconfig-alpha.yaml
+EOF
+
+run <<< "# Wait for the NodeConfig to apply changes to the Kubernetes nodes."
+run <<'EOF'
+kubectl wait --for='condition=Reconciled' --timeout=10m nodeconfigs.scylla.scylladb.com/cluster
+EOF
+# TODO: Extend waiting for the NodeConfig when we fix https://github.com/scylladb/scylla-operator/issues/1557
+
+run <<< "# Install the ScyllaDB local CSI driver that will provision storage for PersistentVolumeClaims."
+run <<< "# The ScyllaDB local CSI driver is configured to use '/mnt/persistent-volumes' directory created by the NodeConfig."
+run <<'EOF'
+kubectl -n local-csi-driver apply --server-side -f=https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/v0.2.0-rc.0/{deploy/kubernetes/local-csi-driver/{00_namespace,10_csidriver,10_driver_serviceaccount,10_provisioner_clusterrole,20_provisioner_clusterrolebinding},example/storageclass_xfs}.yaml
```

Comment:
should we switch to v0.2.0?

Reply:
I can do that, but I'll wait for confirmation that everything else is ok, so I avoid rerecording it again